src/Frontend/src/css/site.scss

```diff
@@ -127,102 +101,97 @@ button.accept-policy {
     background-color: #0d1012;
     height: 100px;
     padding: 1rem;
+    /* Up to xs Breakpoint */
+    @media screen and (max-width: 575.98px) {
+        height: 85px;
+    }
 }
 
 .collapse-wrapper {
     display: flex;
     justify-content: end;
-}
-
-    .collapse-wrapper > div {
+    > div {
         width: 400px;
+        /* Up to sm Breakpoint */
+        @media screen and (max-width: 767.98px) {
+            width: 100%;
+        }
     }
+}
 
 .site-nav-desktop {
     gap: 2rem;
+    .btn-menuburger {
+        background-color: #666666;
+        font-size: 1.75rem;
+        height: 100%;
+        padding: 0.25rem;
+        border: none;
+        &:hover,
+        &:focus {
+            background-color: #525252;
+            color:rgba(0, 0, 0, 0);
+        }
+        &:not(.collapsed) {
+            box-shadow: 0 0 0 0.25rem !important;
+            color: #F9D040;
+        }
+        i {
+            position: relative;
+            top: -12px;
+        }
+        /* Up to sm Breakpoint */
+        @media screen and (max-width: 767.98px) {
+            font-size: 32px;
+        }
+        /* Up to xs Breakpoint */
+        @media screen and (max-width: 575.98px) {
+            height: 85%;
+            font-size: 28px;
+        }
+    }
+    a:last-of-type {
+        margin-right: 1rem;
+    }
+    /* Up to xs Breakpoint */
+    @media screen and (max-width: 575.98px) {
+        &.navbar {
+            padding: 0;
+        }
+    }
 }
 
 nav a {
     color: #FFF;
     font-size: 2rem;
     text-decoration: none;
     transition: color .15s ease;
-}
-
-    nav a:hover,
-    nav a:focus {
+    &:hover,
+    &:focus {
         color: #F9D040;
     }
-
-.site-nav-desktop .btn-menuburger {
-    background-color: #666666;
-    font-size: 1.75rem;
-    height: 100%;
-    padding: 0.25rem;
-    border: none;
 }
 
-    .site-nav-desktop .btn-menuburger:hover,
-    .site-nav-desktop .btn-menuburger:focus {
-        background-color: #525252;
-        color: #F9D040;
-        border: none;
-    }
-
-    .site-nav-desktop .btn-menuburger i {
-        position: relative;
-        top: -12px;
-    }
-
 .navbar-toggler-icon {
     background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(0, 0, 0, 0.5)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E")
 }
 
-.site-nav-desktop .last-nav {
-    margin-right: 1rem;
-}
-
 .site-logo {
     max-width: 240px;
-}
-
-.site-logo img {
-    height: 100%;
-}
-
-/* Up to md Breakpoint */
-@media screen and (max-width: 991.98px) {
-    
+    img {
+        height: 100%;
+    }
 }
 
 /* Up to sm Breakpoint */
 @media screen and (max-width: 767.98px) {
-    
-
-    .site-nav-desktop .btn-menuburger {
-        font-size: 32px;
-    }
-
-    .collapse-wrapper > div {
+    #navbarToggleExternalContent {
```

Comment:
Please uncomment the test for this fix at [tests/Layout.cs](https://github.com/SRCStats/SRCStats/blob/f28cb7c41b4071044b4fbafadc46ee6e05b183d7/tests/Layout.cs#L165-L167).